src/main/java/duke/Task.java

```diff
@@ -0,0 +1,120 @@
+package duke;
+
+import java.io.Serializable;
+
+/*
+ * Abstract super class of all Tasks.
```

Comment:
Perhaps you could add the properties of the Task class here as explanation in your documentation
eg. @param task, @param isComplete